joblib/memory.py

```diff
@@ -818,17 +818,18 @@ def __init__(self, location=None, backend='local', cachedir=None,
         if cachedir is not None:
             if location is None:
                 warnings.warn("cachedir option is deprecated since version "
-                              "0.10 and will be removed after version 0.12.\n"
+                              "0.11 and will be removed after version 0.13.\n"
```

Comment:
I think this should be 0.12 -> 0.14, since 0.11 has already been released. "after version 0.13" is not very clear maybe just say "in version 0.13"

You may have done it already, but please check other places for similar oversights.

Reply:
Done.